src/effects/specialeffectchainslots.cpp

```diff
@@ -0,0 +1,45 @@
+#include "effects/specialeffectchainslots.h"
+
+#include "effects/effectchainslot.h"
+
+QuickEffectChainSlot::QuickEffectChainSlot(const QString& group,
+                                 		   unsigned int iChainNumber,
+                                 		   EffectsManager* pEffectsManager,
+                                 		   const QString& id)
+		: EffectChainSlot(QString("[QuickEffectRack1_%1]").arg(group),
+		  				  iChainNumber, pEffectsManager,
+		  				  SignalProcessingStage::Prefader, false,
+		  				  QString("[QuickEffectRack1_%1]").arg(group)) {
+	// setMix(1.0);
+	// registerInputChannel();
```

Comment:
I think you need to pass a ChannelHandleAndGroup to these EffectChainSlot subclass constructors from EffectsManager. You'll have to call `enableInputChannel` too.